etc/profile-a-l/journal-viewer.profile

```diff
@@ -0,0 +1,66 @@
+# Firejail profile for journal-viewer
+# Description: Visualize systemd logs
+# This file is overwritten after every install/update
+# Persistent local customizations
+include journal-viewer.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.cache/journal-viewer
+noblacklist ${HOME}/.local/share/com.vmingueza.journal-viewer
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.cache/journal-viewer
+mkdir ${HOME}/.local/share/com.vmingueza.journal-viewer
+whitelist ${HOME}/.cache/journal-viewer
+whitelist ${HOME}/.local/share/com.vmingueza.journal-viewer
+whitelist /run/log/journal
+whitelist /var/log/journal
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+net none
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix
+seccomp
+seccomp.block-secondary
+tracelog
+
+disable-mnt
+private
```

Comment:
Are you sure that whitelisting in `${HOME}` works with `private`?

Reply:
Great catch! Originally designed the profile without thinking of adding `private`. That dawned on me much later. I've taken out the ${HOME} whitelisting, also including whitelist-common.inc, which does the same thing.

With that regained awareness I've checked our profiles. Several that do use `private` also include `whitelist-common.inc`. I'm preparing a separate PR for that.

Thank you mucho!

Reply:
> Great catch!

Thanks.

> I've taken out the ${HOME} whitelisting, also including whitelist-common.inc,
> which does the same thing.

Since this program uses paths in `${HOME}`, wouldn't it make more sense to keep
the whitelisting and drop `private`?

Note that the current `noblacklist` commands probably don't do anything either
when `private` is used.

Reply:
> Since this program uses paths in ${HOME}, wouldn't it make more sense to keep
the whitelisting and drop private?

Yeah, you're right. Was overdoing it there...